src/Compilers/CSharp/Portable/Symbols/SourceEmbeddedAttributeSymbol.cs

```diff
@@ -0,0 +1,177 @@
+
+            _namespace = new MissingNamespaceSymbol(compilation.GlobalNamespace, nameParts[0]);
+            for (int i = 1; i + 1 < nameParts.Length; i++)
+            {
+                _namespace = new MissingNamespaceSymbol(_namespace, nameParts[i]);
+            }
+        }
+
+        public SourceEmbeddedAttributeConstructorSymbol Constructor { get; private set; }
+
+        public override int Arity => 0;
+
+        public override ImmutableArray<TypeParameterSymbol> TypeParameters => ImmutableArray<TypeParameterSymbol>.Empty;
+
+        public override NamedTypeSymbol ConstructedFrom => this;
+
+        public override bool MightContainExtensionMethods => false;
+
+        public override string Name => _name;
+
+        public override IEnumerable<string> MemberNames => _members.Select(member => member.Name);
+
+        public override Accessibility DeclaredAccessibility => Accessibility.Internal;
+
+        public override TypeKind TypeKind => TypeKind.Class;
+
+        public override Symbol ContainingSymbol => _namespace;
+
+        internal override ModuleSymbol ContainingModule => _module;
+
+        public override AssemblySymbol ContainingAssembly => _module.ContainingAssembly;
+
+        public override NamespaceSymbol ContainingNamespace => _namespace;
+
+        public override ImmutableArray<Location> Locations => ImmutableArray<Location>.Empty;
+
+        public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences => ImmutableArray<SyntaxReference>.Empty;
+
+        public override bool IsStatic => false;
+
+        public override bool IsAbstract => false;
+
+        public override bool IsSealed => true;
+
+        internal override bool HasTypeArgumentsCustomModifiers => false;
+
+        internal override ImmutableArray<TypeSymbol> TypeArgumentsNoUseSiteDiagnostics => ImmutableArray<TypeSymbol>.Empty;
+
+        internal override bool MangleName => false;
+
+        internal override bool HasCodeAnalysisEmbeddedAttribute => true;
+
+        internal override bool HasSpecialName => false;
+
+        internal override bool IsComImport => false;
+
+        internal override bool IsWindowsRuntimeImport => false;
+
+        internal override bool ShouldAddWinRTMembers => false;
+
+        internal override bool IsSerializable => false;
+
+        internal override TypeLayout Layout => default(TypeLayout);
+
+        internal override CharSet MarshallingCharSet => DefaultMarshallingCharSet;
+
+        internal override bool HasDeclarativeSecurity => false;
+
+        internal override bool IsInterface => false;
+
+        internal override NamedTypeSymbol BaseTypeNoUseSiteDiagnostics => _baseType;
+
+        internal override ObsoleteAttributeData ObsoleteAttributeData => null;
+
+        public override ImmutableArray<Symbol> GetMembers() => _members;
+
+        public override ImmutableArray<Symbol> GetMembers(string name) => _members.Where(member => member.Name == name).ToImmutableArray();
```

Comment:
>_members.Where(member => member.Name == name).ToImmutableArray() [](start = 74, length = 64)

This could be much simple, we have only one member with predefined name.